spreadsheet/chapter.md

```diff
@@ -190,9 +190,9 @@ Finally, we close the `ng-repeat` loop in the column level with `</td>`, close t
 
 The `main.js` file defines the `500lines` module and its `Spreadsheet` controller function, as required by the `<body>` element in `index.html`.
 
-As the bridge between the HTML view and the background worker, it has four purposes:
+As the bridge between the HTML view and the background worker, it has four tasks:
```

Comment:
If I left this as "purposes", we would rewrite the bullet points as "To define the dimensions...", "To provide event handlers..." etc. A "purpose" is always "to do" something. However, if we call them "tasks", we can just use the verbs as written.